core/src/main/kotlin/fr/sncf/osrd/stdcm/OccupancySegment.kt

```diff
@@ -1,14 +1,31 @@
 package fr.sncf.osrd.stdcm
 
+import fr.sncf.osrd.sim_infra.api.BlockId
 import fr.sncf.osrd.utils.units.Distance
 
+interface Segment {
+    val timeStart: Double
+    val timeEnd: Double
+    val distanceStart: Distance
+    val distanceEnd: Distance
+}
+
```

Comment:
I don't think we need the different interfaces, we could directly make one implement the other.

Or in fact, we *may* only need one, if we put default values for the extra fields we may not have to edit the existing tests

Reply:
By the way, I do agree that the inconvenient is having a `Segment` object in the module's classes as well, instead of `OccupancySegment`. I don't really mind, but it's a question of preference. One would hide the whole test object, but make the classes use a higher interface for no "apparent" reason, while the other would incorporate the test object's properties, for no "apparent" reason in the main module.

Reply:
First proposal won't work.
Second proposal: `OccupancySegment` is also to be only used in tests. Moving forward with this fix :)